### DISCUSSION TITLE
No image in HyperHDR when capture card on Mi TV

### DISCUSSION THREAD

**Discussion by timocbro** [awawa-dev/HyperHDR#793] [Q&A]
Hey,

I am currently trying to set up HyperHDR. 
Its running on a Raspberry Pi 3B. 

The following capture card is in use: 
[https://de.aliexpress.com/item/1005003102675265.html?spm=a2g0o.order_list.order_list_main.65.76775c5feLIsTN&gatewayAdapt=glo2deu](url)

It is plugged in to a USB slot on the Raspi and the HDMI cable connects to my Mi TV. 

Unfortunately, all I am getting is a typical test image if I check the live Video in HyperHDR.

Am i missing something obv? 

Thanks guys!

**Comment by javierpal** [comment]
is Mi TV showing an HDR signal?, can you test with something that is not HDR?

**Reply by timocbro** [reply]
I dont think that it is HDR compatible. 
[https://www.kitele.com/en/tv-xiaomi-mi-tv-4a-32](url)

Could it be an HDCP issue?

**Reply by awawa-dev** [reply]
As @javierpal advises: test it on SDR signal to check if the video setup is correct and the grabber is not damaged. If SDR is working and HDR is not, then maybe HDCP or unsupported resolution like 4k60 HDR which needs to be downscaled first.
